src/pages/[gatewayUID]/node/[nodeId]/details.tsx

```diff
@@ -33,33 +37,198 @@ interface SparrowQueryInterface extends ParsedUrlQuery {
+        setIsTempUpdated(true);
+        console.log("Temp updated! ", nodeTempRef.current);
+      }
+      // check node humidity
+      if (
+        !nodeHumidityRef.current ||
+        nodeHumidityRef.current === node.humidity
+      ) {
+        setIsHumidityUpdated(false);
+        nodeHumidityRef.current = node.humidity;
+        console.log("Humidity not updated", nodeHumidityRef.current);
+      } else {
+        nodeHumidityRef.current = node.humidity;
+        setIsHumidityUpdated(true);
+        console.log("Humidity updated! ", nodeHumidityRef.current);
+      }
+      // check node voltage
+      if (!nodeVoltageRef.current || nodeVoltageRef.current === node.voltage) {
+        setIsVoltageUpdated(false);
+        nodeVoltageRef.current = node.voltage;
+        console.log("Voltage not updated", nodeVoltageRef.current);
+      } else {
+        nodeVoltageRef.current = node.voltage;
+        setIsVoltageUpdated(true);
+        console.log("Voltage updated! ", nodeVoltageRef.current);
+      }
+      // check node pressure
+      if (
+        !nodePressureRef.current ||
+        nodePressureRef.current === node.pressure
+      ) {
+        setIsPressureUpdated(false);
+        nodePressureRef.current = node.pressure;
+        console.log("Pressure not updated", nodePressureRef.current);
+      } else {
+        nodePressureRef.current = node.pressure;
+        setIsPressureUpdated(true);
+        console.log("Pressure updated! ", nodePressureRef.current);
+      }
+      // check node motion
+      if (!nodeMotionRef.current || nodeMotionRef.current === node.count) {
+        setIsMotionUpdated(false);
+        nodeMotionRef.current = node.count;
+        console.log("Count not updated", nodeMotionRef.current);
+      } else {
+        nodeMotionRef.current = node.count;
+        setIsMotionUpdated(true);
+        console.log("Count updated! ", nodeMotionRef.current);
+      }
+    }
+  }, [node]);
+
+  const nodeNameStyle = classNames(styles.sectionTitle, {
+    [styles.highlight]: isNameUpdated,
+  });
+  const nodeTempStyle = classNames(detailsStyles.dataNumber, {
+    [styles.highlight]: isTempUpdated,
+  });
+  const nodeHumidityStyle = classNames(detailsStyles.dataNumber, {
+    [styles.highlight]: isHumidityUpdated,
+  });
+  const nodeVoltageStyle = classNames(detailsStyles.dataNumber, {
+    [styles.highlight]: isVoltageUpdated,
+  });
+  const nodePressureStyle = classNames(detailsStyles.dataNumber, {
+    [styles.highlight]: isPressureUpdated,
+  });
+  const nodeMotionStyle = classNames(detailsStyles.dataNumber, {
+    [styles.highlight]: isMotionUpdated,
+  });
+  // end of the code to make highlighting values a thing
 
   const handleDateRangeChange = async (value: string) => {
     // call this function to force a page update with new chart date range
     await router.replace({
       pathname: `${nodeUrl}`,
       query: { minutesBeforeNow: value },
     });
+    await nodeReadingsRefetch();
```

Comment:
Doesn't feel like the `details.tsx` page should have to know about the fact that someone way down inside a chart component changed a dropdown and now that item needs to re-render. Like, if we're using `useQuery` we should be able to delegate that down to that component and then the site can be simpler because each little piece is capable of generating its own viewModel and updating as needed. (I realize there's still the issue of storing some state, such as default date range, in an appropriate place, like the URL Query, or LocalStorage or wherever - but feels like we could abstract that away.)